### PR TITLE
[skip ci] Updated docs to address 0.7.0 issues

### PR DESCRIPTION
**Pull request by npakrasi**

### CODE REVIEW

**Comment by npakrasi** [comment]
LGTM

File: doc/user_doc/vic_app_dev/container_operations.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
...
 |Docker rmi|[Remove a Docker image](https://docs.docker.com/engine/reference/api/docker_remote_api_v1.22/#remove-an-image)|Yes|
-|Docker run|Composite command of create, start, inspect, attach, rm, resize, wait, kill|Yes. <code>docker run -c</code> and <code>docker run -m</code> parameters are supported.  <br>Container search using prettyname-ID <code>docker run -name</code> is supported. <br> Mapping a random host port to the container when the host port is not specified is supported. <br>Running images from private and custom registries is supported.|
+|Docker run|Composite command of create, start, inspect, attach, rm, resize, wait, kill|Yes. Container search using prettyname-ID `docker run -name` is supported. <br> Mapping a random host port to the container when the host port is not specified is supported. <br>Running images from private and custom registries is supported.<br> `docker run --net=host` is not supported.|
```
Comment (anchal-agrawal) [inline doc/user_doc/vic_app_dev/container_operations.md]: `run --name`

File: doc/user_doc/vic_admin/log_bundles.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
+- Live logs (tail files) allow you to view the absolute current status of how components are running.
+<ul>
+- **Docker Personality** is the interface to Docker. When configured with client certificate security, it reports unauthorized access attempts to the Docker server web page.
+- **Port Layer Service** is the interface to vSphere.
+- **Initialization & watchdog** reports network configuration, component launch status for vic-admin and port layer, and records if they fail and relaunches them if they do. The binary  `vic-init` launches the components and redirects their output to the log files in /var/log/vic/ <br>
+At higher debug levels, the component output is duplicated in that log file, so init.log  includes a superset of the log data.
+- **Admin Server** includes logs for the admin server, may contain processes that failed, and network issues. When configured with client certificate security, it reports unauthorized access attempts to the admin server web page.
 
 Live logs can help you see how any current changes you make might affect the logs. For example, when you try to trouble shoot an issue, you can see if your attempt worked or failed by looking at the live logs.
```
Comment (anchal-agrawal) [inline doc/user_doc/vic_admin/log_bundles.md]: Nitpick - `troubleshoot`

**Comment by stuclem** [comment]
Thanks @anchal-agrawal. I'll merge this (if I can) so that I can pull @npakrasi 's changes into my fork. I'll make your two fixes there, then build the output to publish for 0.7/beta 2 today.

**Comment by stuclem** [comment]
LGTM

**Comment by stuclem** [comment]
:recycle:

**Pull request merged**